modules/compiler/targets/host/source/target.cpp

```diff
@@ -256,14 +256,13 @@ compiler::Result HostTarget::initWithBuiltins(
       // for single precision floating points.
       Features.AddFeature("strict-align", true);
       Features.AddFeature("neonfp", true);
-      Features.AddFeature("neon", true);
+      // We need hardware FMA support which is only available as of VFP4.
+      // VFP4 also includes FP16.
+      Features.AddFeature("vfp4", true);
```

Comment:
Does this force us to only run on aarch64 which supports vfp4? Or is that the norm?

Reply:
This is not only AArch64, no, VFP4 is also available on several 32-bit ARMv7 CPUs.

Reply:
what I mean - is this something guaranteed on aarch64? If so, I'm more than happy to approve.

Reply:
Ah, sorry, misunderstood. Yes, my understanding is that this is always available on ARMv8+ which is the minimum that supports 64-bit.